Add heroku buildpack for PHP (#18271)
DataDog/documentation: content/en/developers/community/libraries.md

* [Heroku Datadog Log Drain][35] written in Nodejs by [Oz][36].
* [Heroku Datadog Log Drain][37] written in Go by [Apiary][38].

> To use the tracer or profiler on Heroku you may use a buildpack.

Reviewer: ```suggestion
To use the PHP tracer or profiler on Heroku, use the following buildpack.
```

> * [Heroku Datadog PHP Tracer/Profiler Buildpack][65] maintained by [SpeedCurve][66].

Reviewer: ```suggestion
* [Heroku Datadog PHP Tracer and Profiler Buildpack][65] maintained by [SpeedCurve][66].
```
